.github/workflows/sdist.yml

```diff
@@ -0,0 +1,56 @@
+name: sdist
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+      - 1.2.x
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    name: sdist test
+    timeout-minutes: 60
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.8'
+
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip setuptools wheel
+        pip install numpy cython python-dateutil pytz hypothesis pytest pytest-xdist
```

Comment:
Creating the sdist shouldn't require cython (and also not the other dependencies of pandas and test dependencies)

Reply:
it does require numpy, see https://github.com/pandas-dev/pandas/issues/39416#issuecomment-861457087 as numpy is a top-level import in setup.py

Reply:
(yeah, sorry, I meant everything that comes after "cython" on the above line)